aut-apps.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+echo 'Install Google Chrome, Git and Visual Studio Code'
+touch /etc/apt/sources.list.d/google-chrome.list
+echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main'>/etc/apt/sources.list.d/google-chrome.list
+wget https://dl.google.com/linux/linux_signing_key.pub
+apt-key add linux_signing_key.pub
+curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+install -o root -g root -m 644 microsoft.gpg /etc/apt/trusted.gpg.d/
```

Comment:
I think it should be possible to pipe the key here as well. Plz check.

Reply:
What key are we talking about?

Reply:
`microsoft.gpg` is a public GPG key

Reply:
If you can't find info about piping, use `mktemp` to create a temporary directory under `/tmp`, download the key there and then clean up after install.

Reply:
I create a temporary directory ```console tmp_dir=$(mktemp -d -t pub_key-XXXXXX)``` and I want to change the path in 11 line of code to the path temporary directory like that ```console install -o root -g root -m 644 microsoft.gpg $tmp_dir``` why it's not working?

Reply:
I don't know what doesn't work for you because you didn't describe what you expected and what you've actually got. Of the top of my head, you've messed up the arguments.